__tests__/app.test.js

```diff
@@ -273,4 +273,42 @@ describe("PATCH /api/articles/:article_id", () => {
         });
     });
   });
+});
+
+describe("DELETE /api/comments/:comment_id", () => {
+
+  describe("204: Success response", () => {
+    test("delete comment based on comment_id and returns an empty body", () => {
+      return request(app)
+        .delete("/api/comments/3")
+        .expect(204)
+        .then((response) => {
+          expect(response.body).toEqual({})
+          return db.query('SELECT * FROM comments WHERE comment_id = 3;')
+        })
+        .then(({ rows }) => {
+          expect(rows.length).toBe(0);
+        })
+    });
```

Comment:
It's actually fine to just have the 204 status code come back here, but no harm in doing the query and double checking!